hooks/notification-hook/README.md

```diff
@@ -98,6 +103,36 @@ To configure a slack notification set the `type` to `slack` and the `endPoint` t
 You can use one of the following default templates:
 * slack-messageCard
 
+### Configuration Of An Email Notification
+
+To configure an email notification set the `type` to `email` and the `endPoint` to point to your env containing your target email address.
+You can use one of the following default templates:
+* email
+
+Additional to this configuration you will have to provide a special smtp configuration URL.
+This config reflects the transporter configuration of nodemailer (See [nodemailer | SMTP Transport](https://nodemailer.com/smtp/)).
+This configuration needs to be specified under `env` in the values yaml.
+The identifier for this config has to be `SMTP_CONFIG`.
+A basic configuration could look like this:
+
+```
+...
+env:
+  - name: SMTP_CONFIG
+    value: "smtp://kaden.lueilwitz48@ethereal.email:pezuHHseu8sS2QGcRS@smtp.ethereal.email/"
```

Comment:
The example looks a bit ... strange 🤔
```suggestion
    value: "smtp://youraccount.yourpassword@ethereal.email:FOOBARXYZ@smtp.ethereal.email/"
```

Reply:
According to the documentation of nodemailer this should be a general connection URL for SMTP users and not just etheral.
https://nodemailer.com/smtp/